setup.py

```diff
@@ -72,8 +73,10 @@ def filter_dependencies(*pkgs):
     "sphinx_inline_tabs",
     "sphinxcontrib-programoutput",
     "sphinxcontrib-mermaid",
+    "plotly",
+    "nbimporter"
```

Comment:
Thanks, fixed on 6d85ee56987f38f69a29dfd1c90b0520d2ced52d

Reply:
It's weird because when i removed this line it worked fine and then when I ran this manually it worked fine:
```
(archai) D:\git\microsoft\archai>pip install nbimporter
Collecting nbimporter
  Downloading nbimporter-0.3.4-py3-none-any.whl (4.9 kB)
Installing collected packages: nbimporter
Successfully installed nbimporter-0.3.4
```

Reply:
But I also get this in algos.ipynb:

![image](https://user-images.githubusercontent.com/18707114/218582777-2888760b-b0c9-4008-b343-e76aee30bf1f.png)

so some requirements are still missing...?  I would expect that eventually we have `getting_started\requirements.txt` or even a pip install in the specific notebook for any local dependencies we need ?

Reply:
Or in this case say "you need pytorch installed" with a link to the pytorch.org setup page, because the pytorch install page lists specific commands needed to match your CUDA install, for example, mine is `conda install pytorch torchvision torchaudio pytorch-cuda=11.7 -c pytorch -c nvidia`...

Reply:
Oh, I see the problem, you added nbimporter to a filter_dependencies call, so nbimporter has to first be added to the "dependencies = [ ]" block on line 18.

Reply:
You resolved this too quickly, docs/getting_started/notebooks/discrete_search/algos.ipynb

Reply:
Looks like you fixed it.